tools/mtmd/mtmd-helper.cpp

```diff
@@ -231,12 +231,13 @@ int32_t mtmd_helper_eval_chunk_single(mtmd_context * ctx,
         while (i < n_tokens) { // split into batches
             text_batch.n_tokens = 0; // clear the batch
             for (; i < n_tokens && text_batch.n_tokens < n_batch; i++) {
+                text_batch.token   [text_batch.n_tokens]    = tokens[i];
+                text_batch.pos     [text_batch.n_tokens]    = n_past++;
+                text_batch.n_seq_id[text_batch.n_tokens]    = 1;
+                text_batch.seq_id  [text_batch.n_tokens][0] = seq_id;
+                text_batch.logits  [text_batch.n_tokens]    = false;
```

Comment:
Make the code looks shorter:

```suggestion
                int32_t j = text_batch.n_tokens;
                text_batch.token   [j]    = tokens[i];
                text_batch.pos     [j]    = n_past++;
                text_batch.n_seq_id[j]    = 1;
                text_batch.seq_id  [j][0] = seq_id;
                text_batch.logits  [j]    = false;
```